dev/test-studio/schema/author.ts

```diff
@@ -117,13 +117,13 @@ export default defineType({
 
   initialValue: () => ({
     name: 'Foo',
-    bestFriend: {_type: 'reference', _ref: 'foo-bar'},
-    image: {
-      _type: 'image',
-      asset: {
-        _ref: 'image-8dcc1391e06e4b4acbdc6bbf2e8c8588d537cbb8-4896x3264-jpg',
-        _type: 'reference',
-      },
-    },
+    // bestFriend: {_type: 'reference', _ref: 'foo-bar'},
```

Comment:
This doesn't allow us to create author documents in other workspaces. Any harm on removing this?

Reply:
Maybe we could add another document for this particular test case? Alternatively, as a quick workaround, we could add _strengthenOnPublish + _weak:false to the initial value, that would allow us to create the reference. Won't work for the image asset though.